api_pagopa.yaml

```diff
@@ -0,0 +1,57 @@
+swagger: "2.0"
+info:
+  version: 1.0.0
+  title: Proxy API
+  description: Mobile and web proxy API gateway.
+host: localhost
+basePath: /api/v1
```

Comment:
question: shouldn't we mount these APIs on a different base path than the other APIs? (could generate confusion)